server/image/image.go

```diff
@@ -0,0 +1,78 @@
+package image
+
+import (
+	"context"
+	imagepkg "github.com/argoproj/argo-cd/v2/pkg/apiclient/image"
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/authn/k8schain"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type server struct {
+	keychain authn.Keychain
+}
+
+func (s *server) GetImage(ctx context.Context, req *imagepkg.GetImageRequest) (*imagepkg.GetImageResponse, error) {
+	log.WithField("image", req.Image).Info("GetImage")
```

Comment:
Would want to truncate the image name before logging.

Reply:
I will delete this line.